.eslintrc.js

```diff
@@ -1,4 +1,18 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 const { createConfig } = require('@openedx/frontend-build');
 
-module.exports = createConfig('eslint');
+module.exports = createConfig(
+  'eslint',
+  {
+    rules: {
+      'jsx-a11y/label-has-associated-control': [2, {
+        controlComponents: ['Input'],
+      }],
+      'template-curly-spacing': 'off',
+      'react-hooks/exhaustive-deps': 'off',
+      'no-restricted-exports': 'off',
+      // There is no reason to disallow this syntax anymore; we don't use regenerator-runtime in new browsers
+      'no-restricted-syntax': 'off',
+    },
```

Comment:
Just curious, not blocking - should these settings go into https://github.com/openedx/eslint-config instead of adding them specifically for this MFE?